codebuild/builder.py

```diff
@@ -143,6 +148,16 @@ def __repr__(self):
             "-DENABLE_SANITIZERS=ON",
         ],
     },
+    'macos': {
+        'architectures': {
+            'x86': {
```

Comment:
Wait, we support 32-bit OSX? Is that even a thing at this point?

Reply:
Hell if I know, but couldn't see a reason not to put it in there

Reply:
won't this double our travis builds?

Reply:
No, because travis builds aren't defined by this script, they're defined by travis.yml.